### ISSUE TITLE
Chatheads incorrect chat placement in 1.20.1 when using ShutUpMojang

### ISSUE THREAD

**Issue by Renkie94** [dzwdz/chat_heads#59]
I am guessing you are already familiar with this issue but chat overlays on the chat heads.

![image](https://github.com/dzwdz/chat_heads/assets/140209942/7598d300-1bcb-40e3-914b-45169088f983)

Here is a latest.log
https://mclo.gs/uy6ruNL

This is when using the YORU modpack

I have narrowed it down to an incompatibility between Chatheads and ShutUpMojang
https://modrinth.com/mod/shutupmojang

![image](https://github.com/dzwdz/chat_heads/assets/140209942/38753721-01a0-4523-97fc-2a16056e9d83)

**Comment by Renkie94** [comment]
If you need more information please ask

**Comment by Fourmisain** [comment]
Hm... I think last time I tried the mod, it caused a crash due to a Mixin conflict, but maybe it was another mod afterall...?
Putting that aside, I opened an issue on their side with a partial solution at least. (This is not fixable on our side.)

Generally I would not recommend mods that remove the insecure chat warnings like ShutUpMojang or Chat Signing Hider (which might have a Chat Heads incompatibility as well).
Instead, try to use NoChatReports and avoid insecure servers if possible - though I'm not your boss.
